modules/ROOT/pages/domain-based-application-cookbook.adoc

```diff
@@ -0,0 +1,154 @@
+= Test Domain-Based Applications
```

Comment:
This topic set is really an extended example illustrating how to write a test suite for an example domain-based application by providing example values and then using them in steps explaining how to write a test for that example.

So I think we need a title and a topic introduction that tells under what circumstances a user might want or need to perform such a test, and then mention that the process is most easily explained by providing an example environment and using it to illustrate the steps required to write?create?perform? the test.

I'm not sure offhand how to word a more specific title, or even if that is necessary, since this is about testing (an example) domain-based application. Hmm... How to Write an MUnit Test Suite for Domain-Based Applications? Example of Writing an MUnit Test Suite....?

But I will try to offer some ideas for text around the two subtopics.